packages/dynamite/dynamite/lib/src/models/openapi/info.dart

```diff
@@ -39,18 +39,20 @@ abstract class Info implements Built<Info, InfoBuilder> {
         ..write(summary);
     }
 
-    final description = this.description;
-    if (description != null && description.isNotEmpty) {
+    final description = formatDescription(this.description);
+    if (description != null) {
       buffer
         ..write('\n\n')
         ..write(description);
+
+      print(description);
```

Comment:
left over?

Reply:
Shouldn't the CI be failing because of this? :thinking:

Reply:
Custom lints are only enabled in flutter projects.
And neon_lints/dart.yaml does not enable the avoid_print rule.

we could migrate everything to the logger package and enable the rule. Feel free to open an issue or PR

Reply:
Hm no let's keep it disabled for now.